.husky/pre-commit

```diff
@@ -0,0 +1,60 @@
+# .husky/pre-commit
+. .husky/common.sh
+. "$(dirname "$0")/husky-user-choice.sh"
+. "$(dirname "$0")/setup-messages.sh"
+
+HUSKY_USER_CONFIG_FILE=".husky/user-config"
+
+if [ ! -f "$HUSKY_USER_CONFIG_FILE" ] || ! grep -q '^ggshield=' "$HUSKY_USER_CONFIG_FILE"; then
+  show_ggshield_setup_message
+  exit 1
+fi
+
+if [ ! -f "$HUSKY_USER_CONFIG_FILE" ] || ! grep -q '^reuse=' "$HUSKY_USER_CONFIG_FILE"; then
+  show_reuse_setup_message
+  exit 1
+fi
```

Comment:
this rather feels like a poor-man solution. Have you considered using env vars instead in order to make it configurable?

Reply:
we agreed to use the config file approach